recipes/lyx/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "lyx" %}
+{% set version = "2.13.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://ftp.lip6.fr/pub/lyx/stable/2.3.x/lyx-2.3.6.1.tar.gz
```

The version doesn't match, this should probably be fixed above and the url be changed to:
```suggestion
  url: https://ftp.lip6.fr/pub/lyx/stable/2.3.x/lyx-{{ version }}.tar.gz
```